zebra-chain/src/parameters/constants.rs

```diff
@@ -15,3 +15,15 @@ pub const SLOW_START_INTERVAL: Height = Height(20_000);
 ///
 /// This calculation is exact, because `SLOW_START_INTERVAL` is divisible by 2.
 pub const SLOW_START_SHIFT: Height = Height(SLOW_START_INTERVAL.0 / 2);
+
+/// Magic numbers used to identify different Zcash networks.
```

Comment:
Should we add a link pointing to where did we got these ?

Reply:
I'm not sure where these are specified (I didn't find it in the protocol specification), but I added a link to where I found it in the zcashd code in https://github.com/ZcashFoundation/zebra/pull/8524/commits/f2bef3a8bfbb8f15f97178eaad04ef66bf7e5469.